plugins/plugin_compile/project_compile.py

```diff
@@ -771,11 +767,11 @@ def build_ios(self):
                 "%s" % 'Debug' if self._mode == 'debug' else 'Release',
                 "-scheme" if use_scheme else "-target",
                 "\"%s\"" % targetName,
-                "%s" % "-arch i386" if self.use_sdk == 'iphonesimulator' else '',
+                "%s" % "-arch \"x86_64\"" if self.use_sdk == 'iphonesimulator' else '',
```

Comment:
the default i386 arch app, can't run on the iOS simulator, so change it to 64. tested on Xcode 9.3, macOS 10.13